packages/nextjs/app/builders/0x5E71Cd371cD497A164da8f97aeb853efD160F5f5/page.tsx

```diff
@@ -0,0 +1,127 @@
+import Image from "next/image";
+import type { NextPage } from "next";
+
+const Profile: NextPage = () => {
+  return (
+    <div className="min-h-screen bg-gray-100 text-gray-800 p-10 flex flex-col items-center">
+      <header className="flex flex-col items-center justify-center w-full">
+        <div className="relative flex flex-col items-center">
+          <Image
+            src="/ch0w-profile.jpg" // Add your profile picture in the public directory
+            alt="aniketch0w profile pic"
+            width={150}
+            height={150}
+            className="rounded-full object-cover border-4"
+            style={{ borderColor: "#8246E5" }} // Update border color
+          />
+          <h1 className="text-4xl font-bold mt-4 text-center">Aniket Chaudhary</h1>
+          <p className="text-xl text-gray-600 mt-2 text-center">Blockchain and Web3 Developer</p>
+        </div>
+      </header>
+
+      <main className="max-w-4xl mx-auto mt-10 space-y-8">
+        <section className="bg-white p-6 shadow-md rounded-lg">
```

Comment:
Since the section code is repeated multiple times, creating a component for it would make your code cleaner. That would enhance readability and reduces redundancy.

Reply:
Created a component for the mentioned section of code : 
![Uploading Screenshot 2024-07-27 174313.png…]()